convertool/core/utils.py

```diff
@@ -142,3 +142,108 @@ def makePathParametersAbsolute(
         output = Path(root_path, output)
 
     return (file, output)
+
+
+def expand_json(json_dict: Dict[str, Any]) -> Dict[str, Any]:
+    """
+    * Helper function used to transform the simplified .json structure
+    * Takes the structure in our .json file and expands it to give
+        our convert function the converter and format information it needs
+
+    Parameters
+    ----------
+    json_dict: Dict[str, Any]
+        The already-loaded .json dict
+    """
+    # this is the dict we'll return in the end
+    expanded_json: dict = {}
+
+    # iterate over all the puids
+    for puid in json_dict:
+        # the entry we'll add to the converted dict
+        expanded_entry: Dict[str, Any] = {}
+
+        # the entry in our json for this puid
+        puid_entry = json_dict[puid]
+
+        # expand json_entry into the required style
+        conv_val = puid_entry["converter"]
+        format_val = puid_entry.get("format_type")
```

Comment:
Når du giver None som default eksplicit de andre steder, synes jeg også du skal gøre det her.
```suggestion
        format_val = puid_entry.get("format_type", None)
```